### DISCUSSION TITLE
Numa aware

### DISCUSSION THREAD

**Discussion by 1vaudevillian1** [oobabooga/text-generation-webui#1133] [Ideas]
I was wondering if a feature request for being numa aware goes here, or another git/hugging. If so could numa aware be added? Running a dual socket system and only a single processor spins up.

**Comment by nullgate** [comment]
It's memory bound, so NUMA won't help.

**Reply by 1vaudevillian1** [reply]
It's only using a single processor though. It does not see numa nodes.
